src/main/java/com/example/carsharingservice/service/UserService.java

```diff
@@ -0,0 +1,14 @@
+package com.example.carsharingservice.service;
+
+import com.example.carsharingservice.model.User;
+import java.util.Optional;
+
+public interface UserService {
+    User add(User user);
+
+    User get(Long id);
+
+    User update(User user);
+
+    Optional<User> findByEmail(String email);
```

Comment:
Add one more method, 
`User updateRole(Long id, Role role)`
And implement it
This would be useful on updating user role endpoint